graphics/opencolorio/Portfile

```diff
@@ -107,7 +114,7 @@ if {${configure.build_arch} in [list ppc ppc64]} {
 # OpenColorIO intentially installs Python module in lib
 # see https://github.com/imageworks/OpenColorIO/blob/15e96c1f579d3640947a5fcb5ec831383cc3956e/src/pyglue/CMakeLists.txt#L85
 
-variant python27 description {Build the Python 2.7 bindings} conflicts python38 python39 python310 python311 {
+variant python27 description {Build the Python 2.7 bindings} conflicts python38 python39 python310 python311 python312 {
```

Comment:
can we just drop variants for EOL Python versions please? Not if it's used elsewhere... but ideally you'd just follow the dependency tree and get rid of them all 😉.

Reply:
I don't think it's used anywhere, removed it.